UI/window-projector.cpp

```diff
@@ -148,6 +154,38 @@ void OBSProjector::SetHideCursor()
 		setCursor(Qt::ArrowCursor);
 }
 
+void OBSProjector::SetHideFrame(bool hideFrame)
+{
+	if (isFullScreen())
+		return;
+
+	// Calculate the current content position.
+	QRect contentBox = geometry();
+
+	// Calculate the current window geometry position.
+	QPoint contentPos = pos();
```

Comment:
The error reported by CI should be fixed... At lines 179-180, `geometry()` should be replaced by `contentBox`.